apps/address-service/domains/common/utils/services/search/AbstractSearchPlugin.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Could you drop a few lines about what lib/framework this plugin is made for?

Reply:
Done